ptbr/tests/test_validation.py

```
@@ -0,0 +1,381 @@
+def test_jsonl_loading():
+    entries = [
+        {"tokenized_text": ["Hello", "world"], "ner": [[0, 1, "Greeting"]]},
+        {"tokenized_text": ["Foo", "bar", "baz"], "ner": [[0, 0, "Name"]]},
+    ]
+    with tempfile.NamedTemporaryFile(mode="w", suffix=".jsonl", delete=False) as f:
+        for e in entries:
+            f.write(json.dumps(e) + "\n")
+        tmp = f.name
+    try:
+        data = load_data(tmp)
+        ok, errs = validate_data(data)
```

Ruff (0.15.1)

[warning] Unpacked variable `errs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)